.travis.yml

```diff
@@ -31,6 +31,17 @@ jobs:
         # submit coverage.txt to codecov.io 
         - bash <(curl -s https://codecov.io/bash)
 
+    - <<: *base-test
+      stage: test
+      go: 1.9
+      script:
+        - ./scripts/oc-cluster.sh
+        - make bin
+        - sudo cp ocdev /usr/bin
+        - oc login -u system:admin
```

Comment:
Is system admin really needed? 
We should use the regular user to run all the tests.